state-sync/src/request_manager.rs

```diff
@@ -132,9 +139,15 @@ impl RequestManager {
             self.peers.insert(peer, PeerInfo::new(true, MAX_SCORE));
         }
         self.update_peer_selection_data();
+
+        Ok(())
     }
 
-    pub fn disable_peer(&mut self, peer: &PeerNetworkId, origin: ConnectionOrigin) {
+    pub fn disable_peer(
+        &mut self,
+        peer: &PeerNetworkId,
+        origin: ConnectionOrigin,
+    ) -> Result<(), Error> {
```

Comment:
I'm not sure I buy the value of making it a result?  There are no errors returned here

Reply:
This is for consistency. The high-level function calling this uses the result to know whether or not to print an error (like every other message handler).

Reply:
I've generally used the return value in the signature to know if something **could** fail or not.  Gets rid of having to do extra `?` or `unwrap()` and easier to trace when you're wondering where an error came from.  But, to each their own.

Reply:
Yeah, that's fair!